plugins/operators/model_evaluation/__init__.py

```
@@ -0,0 +1,1052 @@
+            default=selected_values,
+            required=True,
+            label="",
+            description=(
+                f"Select saved views to get started. {len(selected_values)} selected"
+            ),
+            view=types.AutocompleteView(
+                multiple=True,
+                choices=[types.Choice(value=v, label=v) for v in values],
+                allow_duplicates=False,
+                allow_user_input=False,
+                invalid=True if not selected_values else False,
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)